### ISSUE TITLE
About QuAC results

### ISSUE THREAD

**Issue by ShuangPA** [stevezheng23/xlnet_extension_tf#85]
Hi, thank you so much for the code you share, I run the code for QuAC, but only get F1 score around 60, still have a large gap form the result you mentioned. I tried the same parameters you mentioned, except for batch_size, my machine can only support bs=4. Is there something more I need to pay attention to? Can you give me some advice about this? Thanks! :P

**Comment by stevezheng23** [comment]
Hi @ShuangPA , based on your information, I think batch size = 4 is too small for QuAC task, you would want to consider multi-gpu training (supported by this repo) or gradient accumulation (not supported by this repo), thanks
